models/linkages/R/met2model.LINKAGES.R

```diff
@@ -79,6 +88,10 @@ met2model.LINKAGES <- function(in.path, in.prefix, outfolder, start_date, end_da
     sec <- udunits2::ud.convert(sec, unlist(strsplit(ncin$dim$time$units, " "))[1], "seconds")
     dt <- PEcAn.utils::seconds_in_year(as.numeric(year[i])) / length(sec)
     tstep <- 86400 / dt
+    
+    # adjust vector depending on the time step of data 
+    # assumes evenly-spaced measurements
+    DOY_vec_hr <- c(1, c(32, 60, 91, 121, 152, 182, 213, 244, 274, 305, 335, 365) * as.integer(tstep))
```

Comment:
Does this work for leap years? I feel like there's a lubridate function that can give you this vector (rather than hard coding it) in a way that is leap year sensitive

Reply:
Hi Mike! Thanks for looking this over. This is my first pull request. Do I need to make the above adjustments prior to the branch getting merged?

Reply:
Great question -- if the code already works for leap year correctly then I can pull it as is and you can just take the lubridate thing as a suggestion. If it's going to fail for leap years I'd prefer you use existing functions to handle that problem rather than ending up with a  bunch of ifs and alternative hard-coded vectors